users/tests/tests.py

```diff
@@ -34,9 +44,43 @@ def test_redirection_on_register_successful(self):
 
     def test_register_successful(self):
         response = self.client.post(reverse('users:register'), {'username': 'user_1', 'password1': 'pass1234', 'password2': 'pass1234'}, follow=True)
-        self.assertIn('You successfully created a new account!', response.content)
+        self.assertIn(UserRegistration.success_message, response.content)
 
     def test_register_unsuccessful(self):
         UserFactory(username='existing_user')
         response = self.client.post(reverse('users:register'), {'username': 'existing_user', 'password1': 'pass1234', 'password2': 'pass1234'})
         self.assertIn('A user with that username already exists.', response.content)
+
+
+class UserLoginIntegrationTests(TestCase, SimpleTestCase):
```

Comment:
`SimpleTestCase` is a sub-class of the `TestCase`. You should be able to remove the `TestCase` without it having any impact on the tests.